docs/public/GettingHelp.md

```diff
@@ -0,0 +1,15 @@
+# Getting Help
+
+- Refer to [opentelemetry.io](https://opentelemetry.io/) for general
+  information about OpenTelemetry.
+- Refer to the
+  [OpenTelemetry Dotnet GitHub repository](https://github.com/open-telemetry/opentelemetry-dotnet)
+  for further information and resources related to OpenTelemetry Dotnet.
```

Comment:
```suggestion
  for further information and resources related to OpenTelemetry .NET.
```

Reply:
There are more places that I didn't comment on.

Reply:
Reactive this since other places are not updated.

Reply:
Oh my bad, I hadn't pushed the changes. I'll hold off on resolving comments until I push my change in the future 👍🏻